Assets/_BuildingBlocks/LayerStyles/Scripts/BoolExpression.cs

```diff
@@ -0,0 +1,28 @@
+using System.Runtime.Serialization;
+
+namespace Netherlands3D.LayerStyles
+{
+    [DataContract(Namespace = "https://netherlands3d.eu/schemas/projects/layers/styling/expressions", Name = "Bool")]
+    public class BoolExpression : LiteralExpression
+    {
+        [DataMember(Name = "value")] private bool value;
+
+        public BoolExpression(bool value)
+        {
+            this.value = value;
+        }
+
+        public static BoolExpression True() => new(true);
+        public static BoolExpression False() => new(false);
+
+        public override object Resolve(ExpressionContext context)
+        {
+            return value;
+        }
+
+        public override string ToString()
+        {
+            return this.value ? "true" : "false";
```

Comment:
`this.value.ToString()` kan volgens mij ook gewoon

Reply:
Dat wist ik niet, ik ben teveel andere talen gewend die geen ToString op een boolean hebben :D Dat gezegd hebbende: boolean.ToString() geeft "True" en "False" terug. De conventie in JSON en requirement in XML is om lowercase "true" en "false" terug te geven, dus vanuit een compatibiliteitsoverweging laat ik het zo